README.md

```diff
@@ -87,3 +87,6 @@ Explore this repositories commit history to look at your changes and learn about
 **Revert a commit**
 
 &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;`git revert <commit hash>`
+
+
+&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;`git config credential.helper store`
```

Comment:
I think this might hav ebeen accidental as well
```suggestion

```

Reply:
This part was intentional... for people (including myself) who are tired to type in username and pw every single push

Reply:
Never mind me, this is great!